src/components/PostEditor/PostEditor.connector.js

```diff
@@ -54,6 +56,9 @@ export function mapStateToProps (state, props) {
   if (getRouteParam('action', null, props) === 'edit') {
     post = props.post || presentPost(getPost(state, props))
     editing = !!post || loading
+  } else if (fromPostId) {
+    post = props.post || presentPost(getPost(state, props))
+    post.title = `Copy of ${post.title.slice(0, MAX_TITLE_LENGTH - 8)}`
```

Comment:
I added the props.post option for testing before I understood jest fn mocking -- I can update the test and remove it

Reply:
I removed the `props.post |`